backends/k8s/client.go

```diff
@@ -300,6 +300,9 @@ func populateNodeDetails(kNode *kapiv1.Node, kvps map[string]string) error {
 		kvps[nodeKey+"/network_v4"] = node.BGPIPv4Net.String()
 	}
 
+	// Some empty defaults for ipv6
```

Comment:
I can move it to another PR if we need, but it is a small fix that was caught when running confd in `-onetime` mode, when it sees this key is missing in the `bird6.cfg` template it issues a log to `Fatal` but when run with `-watch` or `-interval` it logs to `error` so it carries on to create the rest of the templates.  This fixes that so it generates all of the templates in `-onetime` mode.